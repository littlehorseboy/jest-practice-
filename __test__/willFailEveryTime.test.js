it('will fail every time', () => {
  const user = {
    createdAt: new Date(),
    id: Math.floor(Math.random() * 20),
    name: 'horse',
  };

  expect(user).toMatchSnapshot();
});
