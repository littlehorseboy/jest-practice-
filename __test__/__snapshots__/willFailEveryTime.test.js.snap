// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will fail every time 1`] = `
Object {
  "createdAt": 2019-03-12T15:14:46.652Z,
  "id": 1,
  "name": "horse",
}
`;
